README.md

```diff
@@ -0,0 +1,30 @@
+# Honeycomb Tracing for Sequel
+
+[![Build Status](https://travis-ci.org/honeycombio/sequel-honeycomb.svg?branch=master)](https://travis-ci.org/honeycombio/sequel-honeycomb)
+[![Gem Version](https://badge.fury.io/rb/sequel-honeycomb.svg)](https://badge.fury.io/rb/sequel-honeycomb)
+
+This package makes it easy to instrument your Sequel database calls in your ruby application to send useful events to [Honeycomb](https://www.honeycomb.io), a service for debugging your software in production.
+- [Usage and Examples](https://docs.honeycomb.io/getting-data-in/beelines/ruby-beeline/)
```

Comment:
This is the only bit that I'm unsure on. It's not specific to this gem but might still be useful for people. We don't really have any other place to send them though

Reply:
Yeah, seems better than nothing for now 👍